src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -90,6 +90,10 @@ function MonacoInstance ({logFileState, changeStateCallback, loadingLogs, logDat
      * @param {object} monaco
      */
     function handleEditorWillMount (monaco) {
+        // "pageSize" will be restored in handleEditorDidMount()
+        //  if loading Monaco with this limit does not cause client OOM
```

Comment:
```suggestion
        // Disable the cached "pageSize" in case it causes a client OOM. If it
        // doesn't, the saved value will be restored in handleEditorDidMount().
```